drivers/gnss/gnss_quectel_lcx6g.c

```diff
@@ -668,6 +674,13 @@ static struct gnss_driver_api gnss_api = {
 	.get_supported_systems = quectel_lcx6g_get_supported_systems,
 };
 
+static void quectel_lcx6g_init_lock(const struct device *dev)
```

Comment:
the wrappers seems a bit unnecessary to be honest, I would drop at least the init one

Reply:
Wrapper has vanished :)